### ISSUE TITLE
Add Compiled  PDF Versions as Download to the README

### ISSUE THREAD

**Issue by LinusDietz** [fs-wiai/LaTeX-Tutorial#6]
As an interested person in learning LaTeX, I want to be able to download the tutorial as a PDF without having to download the sources and compiling them on my own.
Ideally, there would be a web server from where the current PDF can be downloaded.
Even better would be if this version would be updated automatically after each push to  the `master` branch. (Continuous Delivery)

The README should be updated accordingly with a prominent link.

I can help you setting up such a pipeline (similar to e.g., http://builds.jabref.org/) if you are interested.

**Comment by muellermartin** [comment]
Usually we publish the PDF files in the VC course, but having them publicly available would be nice nevertheless.

> Ideally, there would be a web server from where the current PDF can be downloaded.

For the hosting of the releases I would simply use the GitHub release feature.


> Even better would be if this version would be updated automatically after each push to the master branch. (Continuous Delivery)

Agreed. This could be done with a post commit hook. GitHub also provides such a feature. The difficult question is where the build environment should run? Travis CI could be a very easy option as it's free for open source projects and we don't need to maintain the build environment. Alternatively, we could create a "simple" service on our infrastructure which fires up a Docker container with a LaTeX build environment, builds the project and uploads the build (archive) to GitHub.


> I can help you setting up such a pipeline (similar to e.g., http://builds.jabref.org/) if you are interested.

That sounds nice :) Feel free to submit your ideas and I see what we need to do to realize that.

**Comment by LinusDietz** [comment]
Yep, Travis or Circle CI can be used for building, the only question is where to host non-release, builds for download? That cannot be done on the build servers.

Suggestion: An FTP server (VM) on any of the Fachschaft clusters? Can you set that up?

**Comment by muellermartin** [comment]
I don't think non-release builds are an issue, as an editor needs to have a LaTeX toolchain available locally, so every author will have his own development build anyway and for others they are not needed in my opinion.
